src/index.js

```diff
@@ -0,0 +1,174 @@
+//--API--//
+
+function getForecast(coordinates) {
+    console.log(coordinates);
```

Comment:
if it's a "production" code, remove all console.log. they're used for checking if your code is correct, doesn't need to stay there if code works already